dev-docs/bidders/conceptx.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: bidder
+title: Conceptx
+description: Conceptx bidder adapter
+biddercode: conceptx
+gdpr_supported: true
+usp_supported: false
+coppa_supported: false
+schain_supported: false
+floors_supported: true
+media_types: banner
```

Comment:
We always recommend to add `safeframes_ok`: `true` / `false` meta data as well

Reply:
I added it now